src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -199,7 +203,7 @@ export class ReadingsGraphComponent implements OnInit {
         backgroundColor: '#239B56',
         borderColor: '#82E0AA',
       }];
-    } else {
+    } else if (assetReading.length === 1) {
```

Comment:
what is the case here? when length ==1

Reply:
isValidData is not changed in asset-summary.component. Working as it is.

that length check is to check if there is any data in there into dataset.